FrameworkSystem/private/standardLogging/Backend/Backend.py

```diff
@@ -0,0 +1,53 @@
+"""
+Backend wrapper
+"""
+
+__RCSID__ = "$Id$"
+
+from abc import ABCMeta, abstractmethod
```

Comment:
You mean to act just as an interface ? Don't bother doing things like this. Just call it AbstractBackend, and return an error or an exception in the methods ;-)